### PR TITLE
DB-12129 support showing region info in SpliceCK

### PR DESCRIPTION
**Pull request by hatyo**
## Short Description
In this PR we introduce a new command that views information of table regions.

## Long Description
The previous command `regionof` is repurposed to show table region information such as HFiles, byte size, start, stop, host, ... etc (check the example output below for more information), the old command semantics are put in a new command called `conglomof`.

## How to test

Here is an example output of the command:

```
spliceck> regionof splice t1
index     name                                 offline     start     stop     host                HFile count     HFile size     MemStore size     last major compaction
===============================================================================================================================================================================
0         d46dafc2990b31d68982c902e8c1d923     false                 C096     localhost:52747     0               0.0MB          0.0MB             never compacted
1         55a59e79a89b48e86cc2add774453712     false       C096      C2A4     localhost:52747     0               0.0MB          0.0MB             never compacted
2         98ec6c22fece60bccab5f8a23d000dc9     false       C2A4               localhost:52747     1               1.0MB          0.0MB             2021-07-15 18:37:13.783
```

### CODE REVIEW

**Comment by hatyo** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/3385/

**Comment by hatyo** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/3395/

**Review by martinrupp — APPROVED** [review]

File: splice_ck/src/main/java/com/splicemachine/ck/Constants.java [hunk 1]
```diff
@@ -69,4 +69,15 @@
 
     public static final String HBASE_CONFIGURATION_ZOOKEEPER_QUORUM = "hbase.zookeeper.quorum";
     public static final String HBASE_CONFIGURATION_ZOOKEEPER_CLIENTPORT = "hbase.zookeeper.property.clientPort";
+
+    public final static String TBL_REGION_COL0 = "index";
```
Comment (ipraznik-splice) [inline splice_ck/src/main/java/com/splicemachine/ck/Constants.java:73]: Tabular(SortHint sortHint, String... headers) can accept an array as headers. Maybe better to define column names as 
String TBL_REGION_COLS[] = {"index", "name", "offline", ..} to have it shorter, as constant names for every column not important here.

**Review by ipraznik-splice — APPROVED** [review]

**Comment by hatyo** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/3401/

**Review by carolp-503 — APPROVED** [review]
+1

**Comment by hatyo** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST SUCCEEDED +1

**Pull request closed without merging**